src/ClusterTSnePlot.js

```diff
@@ -100,6 +100,11 @@ const ClusterTSnePlot = (props) => {
             })
           })
 
+        // Trick Highcharts into thinking the point is in the bottom half of the chart, so that the tooltip
+        // is displayed below the point
+        this.point.negative = true
+        tooltip.updatePosition(this.point)
```

Comment:
It turns out that once `this.point.negative` is set to true, there’s no need to call `tooltip.updatePosition` since it will be called implicitly at the end of `tooltip.formatter`.

I also think that having `this.point.negative` the first line of the function separates better presentation and content-related code, but that’s a very minor point.